client/asset/btc/btc.go

```diff
@@ -576,6 +577,13 @@ func (btc *ExchangeWallet) Connect(ctx context.Context) (*sync.WaitGroup, error)
 	if err != nil {
 		return nil, fmt.Errorf("error initializing best block for %s: %w", btc.symbol, err)
 	}
+	// Check for method unkown error for feeRate method.
+	_, err = btc.estimateFee(ctx, btc.node.requester, 1)
+	var rpcErr *dcrjson.RPCError
+	if errors.As(err, &rpcErr) && rpcErr.Code == dcrjson.ErrRPCMethodNotFound.Code {
```

Comment:
Sorry to circle back on this, but I believe we should be using the constants in btcjson, although clearly we'd need to assert with `dcrjson.RPCError`.  Either that or we just make a package level const with the expected value of `ErrRPCMethodNotFound.Code` to spare us importing btcjson just for one number.  Just that the error codes between dcr and btc's rpc servers aren't necessarily the same, and I've noted commits with bitcoin where they've actually changed the codes and I'm sure we didn't change them too.